test/e2e/lib/components/site-editor-component.js

```diff
@@ -156,4 +194,32 @@ export default class SiteEditorComponent extends AsyncBaseContainer {
 			)
 		);
 	}
+
+	async dismissNotices() {
+		const locator = By.css( '.components-snackbar[aria-label="Dismiss this notice"]' );
+		const notices = await this.driver.findElements( locator );
+		await Promise.all( notices.map( ( notice ) => notice.click() ) );
+		await driverHelper.waitUntilElementNotLocated( this.driver, locator );
+	}
+
+	async insertBlockOrPatternViaBlockAppender( name, container = 'Group' ) {
+		const containerBlockId = await this.addBlock( container );
+		await this.runInCanvas( async () => {
+			await driverHelper.clickWhenClickable(
+				this.driver,
+				By.css( `#${ containerBlockId } .block-editor-button-block-appender` )
+			);
+		} );
+		await driverHelper.setWhenSettable(
+			this.driver,
+			By.css( '.block-editor-inserter__quick-inserter .block-editor-inserter__search-input' ),
+			name
+		);
+		await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css(
+				'.block-editor-inserter__quick-inserter .block-editor-block-types-list__item, .block-editor-inserter__quick-inserter .block-editor-block-patterns-list__item'
```

Comment:
It's generally a good practice to assign selectors/locators to constants. Naming them helps understand what is actually being clicked.